docs/features/techdocs/how-to-guides.md

```diff
@@ -483,6 +483,8 @@ plugins:
   - techdocs-core
 ```
 
+note: 
```

Comment:
I'm not sure why this was added here, might have been by mistake?

Reply:
Yes, it is a mistake. Sorry!

Reply:
Removed the line accordingly